src/components/blocks/profileBlock.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FormEvent, useState } from "react";
+import Block from "@/components/blocks/block";
+import PictureChoose from "@/elements/pictureChoose/pictureChoose";
+import InputText from "@/elements/inputText/inputText";
+import InputTextArea from "@/elements/inputText/inputTextArea";
+import styles from "./blockForm.module.scss";
+import { formFieldByName } from "@/data/formFields";
+import SignButton from "@/elements/signButton/signButton";
+import ChangePassword from "@/pages/users/changePassword";
+
+interface IProfileBlockProps {
+  blockName: string;
+}
+
+const useFields: () => [
+  string,
+  React.Dispatch<React.SetStateAction<string>>,
+  string,
+  React.Dispatch<React.SetStateAction<string>>
+] = () => {
+  const [userName, setUserName] = useState("");
+  const [description, setDescription] = useState("");
+
+  return [userName, setUserName, description, setDescription];
+};
+
+const ProfileBlock = (props: IProfileBlockProps) => {
+  const [userName, setUserName, description, setDescription] = useFields();
+
```

Comment:
you dont need custom hook for this case. just use ` const [userName, setUserName] = useState("");
  const [description, setDescription] = useState("");` 
  
  If you want to create custom hooks, you must create separate folder for them and use export/import

Reply:
made as you said